cli/BUILD

```diff
@@ -22,6 +22,7 @@ java_binary(
 kt_jvm_library(
     name = "cli-lib",
     srcs = glob(["src/main/kotlin/**/*.kt"]),
+    resources = ["//:.bazelversion"],
```

Comment:
This looks good. We need to use a different version file then this one since this one controls the version of Bazel that is running, not the version of bazel-diff

Reply:
Oh oops, currently when the binary is published, how is the version passed in? I see that since it follows semvar, it seems like it might be provided from the pipeline itself?

Reply:
ah looks like it picks it up from tags.

Reply:
@tinder-maxwellelliott correct my assumption above, updated the PR to utilize bazel stamping to embed the version tag and read it directly from the binary resources instead.